package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "nodemon src/index.js",
     "build": "sucrase ./src -d ./dist --transforms imports",
-    "test": "mocha --require sucrase/register ./test/**/*.js"
+    "test": "NODE_ENV=test mocha --require sucrase/register ./test/**/*.js"
```

Comment:
Since we are using `dotenv` this value should be set on `.env`

Reply:
I've removed it from package.json and I did write how to run the tests on README file.

Reply:
The common convention is to set NODE_ENV in your build script and not in .env file so @DronePT was right to set in `package.json`. If you set in ,env you'll have to remeber to alter this variable every time that you wants to change the environment and possible having some side effects.

Reply:
Could you please go back to your original version @DronePT?

Reply:
@joviane setting NODE_ENV on test script is what I'm used to do also. I've reverted the branch to the last commit. 

Thank you